app/assets/stylesheets/settings/README.md

```diff
@@ -1,3 +1,5 @@
 # Settings
 
 Used with preprocessors and contain font, colors definitions, etc
+
+1. spacing.scss - contains consistent incremented spacing variables (derived from the base line height in Pugin) to be used for consistent white spacing, margins, padding, etc across Parliament web services.
```

Comment:
This is slightly pedantic but if it really offends and you feel the consumers of this 'readme' will misunderstand the file I am referencing then it can be changed.